### PR TITLE
Rename to `open-webui`

### LINKED ISSUE #767: k8s: rename to open-webui

### PR DESCRIPTION
**Pull request by bjornjorgensen**
fix #767

### CODE REVIEW

**Comment by bjornjorgensen** [comment]
hmm.. will you have everything like ollama-webui-service renamed to open-webui--service?

**Comment by tjbck** [comment]
@bjornjorgensen Thanks for the PR! As long as it won't cause any issues we should!

File: kubernetes/helm/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
 name: open-webui
-description: "Ollama Web UI: A User-Friendly Web Interface for Chat Interactions 👋"
+description: "open-webui UI: A User-Friendly Web Interface for Chat Interactions 👋"
 version: 1.0.0
-icon: https://raw.githubusercontent.com/ollama-webui/ollama-webui/main/static/favicon.png
+icon: https://github.com/open-webui/open-webui/blob/main/static/favicon.png
```
Comment (justinh-rahb) [inline kubernetes/helm/Chart.yaml:5]: This won't properly resolve to an image, need to change it to the raw CDN address.

File: kubernetes/helm/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: ollama-webui
-description: "Ollama Web UI: A User-Friendly Web Interface for Chat Interactions 👋"
+name: open-webui
+description: "open-webui UI: A User-Friendly Web Interface for Chat Interactions 👋"
```
Comment (bjornjorgensen) [inline kubernetes/helm/Chart.yaml:3]: is it open-webui or Open webUI or ?

**Comment by bjornjorgensen** [comment]
yes this needs a litle bit more work..

File: kubernetes/helm/Chart.yaml (hunk 2 above)
Comment (justinh-rahb) [reply kubernetes/helm/Chart.yaml:3]: > is it open-webui or Open webUI or ?

Open WebUI, Open-WebUI, open-webui in order of preference/as appropriate for application.

**Comment by bjornjorgensen** [comment]
hmm.. why ain't this being splitte to into two? 

this is how it is now 
image

but there should be one namespace for open-webui and one for ollama and a service between them

**Comment by jnkstr** [comment]
@bjornjorgensen this is normal. Was the same before the renaming.

**Comment by jnkstr** [comment]
> @jnkstr yes, but it ain't best practice and it looks messy 

Yes k8s is not that much used in this project for now i guess. 

Feel free to change it! We are also deploying with k8s.

**Comment by justinh-rahb** [comment]
> Yes k8s is not that much used in this project for now i guess.
> 

At least not by the maintainer or any other major contributors up to now besides yourself I think @jnkstr.

**Comment by jnkstr** [comment]
Not sure. Maybe @dnviti and @braveokafor are using k8s.

**Comment by braveokafor** [comment]
I agree, the current Helm/K8s implementations are somewhat messy and warrant a rewrite; they were rushed the first time around.

@bjornjorgensen Is the `Open WebUI` change a step towards a more "LiteLLM style" backend-agnostic future? I'm trying to understand why we require two separate namespaces.

I placed WebUI and Ollama in separate folders in [my chart](https://github.com/braveokafor/ollama-webui-helm/tree/main/charts/ollama-webui-helm/templates).

> My chart is older than the one in this repo, but I use Bitnami common charts, I wasn't sure how the community would feel about an added dependency.

**Comment by bjornjorgensen** [comment]
Now I have tested the helm files.

to have support for tls I have edit webui-ingress.yaml like this
```
{{- if .Values.webui.ingress.enabled }}
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: open-webui-ingress
  namespace: {{ .Values.namespace }}
  {{- if .Values.webui.ingress.annotations }}
  annotations:
    {{ toYaml .Values.webui.ingress.annotations | trimSuffix "\n" | indent 4 }}
  {{- end }}
spec:
  {{- if and .Values.webui.ingress.tls (gt (len .Values.webui.ingress.tls) 0) }}
  tls:
  {{- range .Values.webui.ingress.tls }}
  - hosts:
    - {{ .hosts | first }}
    secretName: {{ .secretName }}
  {{- end }}
  {{- end }}
  rules:
  - host: {{ .Values.webui.ingress.host }}
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: open-webui-service
            port:
              number: {{ .Values.webui.servicePort }}
{{- end }}
```
and values.yaml like this
```

namespace: open-webui

ollama:
  gpu:
    enabled: false
  replicaCount: 1
  image: ollama/ollama:latest
  imagePullPolicy: Always
  servicePort: 11434
  resources:
    limits:
      cpu: "2000m"
      memory: "2Gi"
  volumeSize: 1Gi
  service:
    type: ClusterIP

webui:
  replicaCount: 1
  image: ghcr.io/open-webui/open-webui:main
  imagePullPolicy: Always
  servicePort: 8080
  resources:
    limits:
      cpu: "500m"
      memory: "500Mi"
  volumeSize: 10Gi
  ingress:
    enabled: true
    annotations:
      cert-manager.io/cluster-issuer: "letsencrypt-prod"
    host: open-chat.bjhjemme.duckdns.org
    tls:
    - hosts:
      - open-chat.bjhjemme.duckdns.org
      secretName: open-chat-tls-secret
  service:
    type: NodePort

```

and it works :)

**Comment by duhow** [comment]
I'm open to contribute in improving this Helm Chart, but I'll wait until all your changes are done, just to avoid conflicts.

**Comment by bjornjorgensen** [comment]
@tjbck I think this is ready now.

**Comment by tjbck** [comment]
Great, Thanks for the hard work everyone!

**Pull request merged**